packages/dds/tree/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -490,14 +498,15 @@ export function uniformChunkFromCursor(
 	shape: TreeShape,
 	maxTopLevelLength: number,
 	skipLastNavigation: boolean,
+	idCompressor?: IIdCompressor
```

Comment:
this function needs documentation to cover what it means to provide an id compressor not. insertValues should also get those docs (or link the ones here)

Reply:
Updated `insertValues` and `uniformChunkFromCursor` functions to document what it means to provide an id compressor